src/runtime-rs/Makefile

```diff
@@ -167,7 +167,7 @@ DEFVFIOMODE := guest-kernel
 ##VAR DEFSANDBOXCGROUPONLY=<bool> Default cgroup model
 DEFSANDBOXCGROUPONLY ?= false
 DEFSTATICRESOURCEMGMT ?= false
```

Comment:
CLH depends on the `DEFSTATICRESOURCEMGMT`. 
IMO, I'm afraid that CLH with the configuration will not work well, if it's set with false.

Reply:
fixed

Reply:
Just for the sake of making things clear.

Cloud Hypervisor itself could be set to `false` without any issues, as Cloud Hypervisor supports memory / cpu hotplug.  However, when we deal with TEEs (TDX, in our case), we need to have it set to `true` as TDX doesn't support memory / cpu hotplug.

For now, TDX is not supported on Cloud Hypervisor, and even when TDX will be supported, we'll need a new configuration file for it.  So, it can be set to `false`  here without issues.

Reply:
fixed

Reply:
cc @jodh-intel

Reply:
@Tim-0731-Hzt - To avoid breaking Cloud Hypervisor, how about adding a `CLH_DEFSTATICRESOURCEMGMT` option?

Reply:
Hi @jodh-intel @amshinde Could you please check if the kata pod running ok with CLH(runtime-rs) when the `static_resource_mgmt` is set the default `false`? I'm afraid that it won't work well with the un-implement of `resize_memory` in CLH.

Reply:
CC @Tim-0731-Hzt [related issue](https://github.com/kata-containers/kata-containers/issues/8784)